deployment/gke/deployment_manager_configs/cluster-kubeflow.yaml

```diff
@@ -31,10 +31,10 @@ resources:
     zone: SET_THE_ZONE
     # "1.X": picks the highest valid patch+gke.N patch in the 1.X version
     # https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.zones.clusters
-    cluster-version: "1.10"
+    cluster-version: "1.11"
     # Set this to v1beta1 to use beta features such as private clusterss
     # and the Kubernetes stackdriver agents.
-    gkeApiVersion: v1
+    gkeApiVersion: SET_GKE_API_VERSION
```

Comment:
This is outside the scope of this PR. The fact that we can't set meaningful default values because we rely on SED to do substitution is not good.

It would be better to use a tool like jsonnet or yq to allow more powerful substitution. We've avoided that so far because it would introduce additional dependencies.

We should be able to do that once we move to replacing kfctl.sh with a go binary.

Reply:
Ack